db/migrate/20160310030824_create_pieces.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePieces < ActiveRecord::Migration
+  def change
+    create_table :pieces do |t|
+    	t.integer :x_coordinate
+    	t.integer :y_coordinate
+    	t.string :color
+    	t.boolean :taken
+    	t.integer :player_id
```

Comment:
Probably no need to connect the player with the piece. Could do it via the color.